src/Components/Facility/UpdateMinQuantity.tsx

```diff
@@ -149,6 +148,7 @@ export const UpdateMinQuantity = (props: any) => {
                     type="string"
                     value={data}
                     errors=""
+                    disabled={true}
```

Comment:
If we are setting this as disabled always, why should this be an input field, can we design it better?